source/_components/sensor.template.markdown

```diff
@@ -258,6 +258,9 @@ This template contains no entities that will trigger an update, so we add an `en
 {% raw %}
 ```yaml
 sensor:
+- platform: time_date
+  display_options:
+  - date
```

Comment:
There is no need to add the sensor as it's mentioned in the intro for the section that a `time_date` sensor is used. It it's not clear then the intro should be updated. 

None of the other sample is showing the source entity. Thus if one is not using the `sun` component the sample will not work.

Reply:
Thanks for the feedback! I adjusted my proposal to be more in line with your commentary.

I do think _some_ mention is warranted. It would've at least saved me some time, and I'm inclined to believe that other folks would run into the same confusion.

Reply:
I don't think that the latest change makes it clearer... "may need to"? So when in that case?
Also, I'm not sure if "registered" is the right term here.